Dockerfile

```
@@ -0,0 +1,19 @@
+# Dockerfile
+FROM php:8.3.2-cli
+
+RUN apt-get update -y \
+  && apt-get install -y libmcrypt-dev \
+  && apt-get clean \
+	&& docker-php-ext-install bcmath \
+  && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*
+
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+WORKDIR /app
+COPY . /app
+
+RUN composer install
+RUN php artisan key:generate
+
+EXPOSE 8000
+CMD php artisan serve --host=0.0.0.0 --port=8000
```

After https://github.com/minvws/nl-uzipoc-php-laravel/pull/22 is merged.

We need to copy the startup script to something like `/scripts/startup.sh`.
And set the CMD to:
```suggestion
CMD /scripts/startup.sh
```